drivers/hmis/app/graphql/mutations/update_project.rb

```diff
@@ -20,17 +20,20 @@ def resolve(id:, input:, confirmed:)
       response
     end
 
-    def create_warnings(project, input)
+    def create_errors(project, input)
       return [] unless project.operating_end_date.blank? && input.operating_end_date.present?
 
+      errors = HmisErrors::Errors.new
+
       funder_count = project.funders.where(end_date: nil).count
       inventory_count = project.inventories.where(inventory_end_date: nil).count
       open_enrollments = Hmis::Hud::Enrollment.open_on_date.in_project_including_wip(project.id, project.project_id)
```

Comment:
Should this be checking the OperatingEndDate instead of today?
```suggestion
      open_enrollments = Hmis::Hud::Enrollment.open_on_date(project.operating_end_date).in_project_including_wip(project.id, project.project_id)
```

Reply:
I don't know, should it? This is for the warning when you are closing a project, saying "There are X open enrollments."

If you are setting the end date to Feb 1st and there is an enrollment that was closed yesterday (Feb 7) do you need to be warned about that enrollment? Basically "1 enrollment was still open on the selected project end date"..

Reply:
I think it should warn if any enrollment is open on the close date.  If that date is in the past, we'll still want to know there were open enrollments, and if it's in the future, we should allow it, but warn if there are enrollments that haven't been closed by that future date.

Reply:
Okay. I will change the logic in a separate PR so it can be more easily tracked with PT.